images/kubekins-e2e/Dockerfile

```diff
@@ -17,7 +17,7 @@
 
 ARG OLD_BAZEL_VERSION
 FROM launcher.gcr.io/google/bazel:${OLD_BAZEL_VERSION} as old
-FROM gcr.io/k8s-testimages/bootstrap:v20200728-3f8a003
+FROM gcr.io/k8s-prow/bootstrap:v20200729-ee80a02
```

Comment:
The post-submit job listed k8s-prow for the image, not sure if there is a separate process for pushing to k8s-testimages, or why it would be preferred over k8s-prow here.